### ISSUE TITLE
[BUG] Crash every second run

### ISSUE THREAD

**Issue by ViRb3** [AnalogJ/scrutiny#278]
**Describe the bug**
Every time I start scrutiny, it crashes. When I restart it, it always works, until I restart it again. I have no idea why this is happening.

**Log Files**
https://paste.gg/p/anonymous/eb6fc8ca71244fb6bc000d14388aa8e8

**System**
- Ubuntu 22.04 arm64 (Raspberry Pi 4)
- Docker Compose with official container:
  - `ghcr.io/analogj/scrutiny:master-omnibus@sha256:f79b6c118c85ac81164ee54b28d023f6da8536e7ea2be2f207c81c512f753c1e`

**Comment by ViRb3** [comment]
Superseded by #281

**Comment by ViRb3** [comment]
Issue returned after #281 was fixed.

**Comment by ViRb3** [comment]
Fixed in #281.
